lib/codecs/adts.js

```diff
@@ -64,7 +64,7 @@ AdtsStream = function(handlePartialSegments) {
 
     // Prepend any data in the buffer to the input data so that we can parse
     // aac frames the cross a PES packet boundary
-    if (buffer) {
+    if (buffer && buffer.length) {
```

Comment:
Buffer is no longer set to undefined if we don't have bytes left, I was torn on initializing it to an empty uint8 array and always concating with buffer to make all of this code simpler.